lisdf/parsing/parse_pddl.py

```diff
@@ -0,0 +1,47 @@
+from typing import Dict, List, Set, Tuple
+
+import pddlpy
+
+
+def parse_pddl(
+    domain_file_path: str, problem_file_path: str
+) -> Tuple[List[List[str]], List[List[str]], Dict[str, Set[str]]]:
```

Comment:
I personally would prefer if this were wrapped in a dataclass/namedtuple or something. That's probably a bit too much effort for now though.

Could we make the typing a bit more readable? e.g.

```
Predicate = str
Type = str
Object = str

# First element in an atom is the predicate name, the rest are argument objects
Atom = List[Union[Predicate, Object]]
State = List[Atom]

-> Tuple[State, State, Dict[Type, Set[Object]]
```